doc/installation_and_administration_guide/hw_requirements.md

```diff
@@ -0,0 +1,2 @@
+* RAM: 1 GB, specially if abusing of the batching mechanism.
+* HDD: A few GB may be enough unless the channel types are configured as FileChannel type.
```

Comment:
Fixed in <https://github.com/telefonicaid/fiware-cygnus/commit/69bad2f614891f36366db42e5b3636c2d0b591d0>